### PR TITLE
New profile text

### PR DESCRIPTION
**Pull request by anokhikastiaIBM**
Create a new profile text #1123

When you click the + icon on the DATA SETS tree to create a new profile, then:
Add should be lowercase
Based on the type of the tree that we are adding a file to, the message is changed: Dataset, Job or USS.

## Release Notes
The following files have been changed:
Profiles.ts, Profiles.unit.test.ts, mvsCommandHandler.unit.test.ts, tsoCommandHandler.unit.test.ts, DatasetTree.unit.test.ts and actions.unit.test.ts.
https://github.com/zowe/vscode-extension-for-zowe/issues/1123

Milestone: 1.20.0

Changelog: Based on the type of the tree (Dataset, Job or USS) that we are adding a file to, I change the message that is being displayed is changed. The new message also fits in the box and is completely visible to the customer.

[Zowe 1123.pdf](https://github.com/zowe/vscode-extension-for-zowe/files/7228620/Zowe.1123.pdf)
## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by rudyflores — COMMENTED** [review]
@anokhikastiaIBM Everything seems to be good with your PR! I have tested both unit/integration tests and they all seem to be passing. As for the feature itself I was curious whether the description on the input box was originally overflowing and wasn't legible as it is currently with your changes. If it wasn't maybe we could find a shorter way to describe what is in that textbox to prevent the text overflow.

Here's an example of what i'm describing:
Screen Shot 2021-09-20 at 12 16 38 PM

**Review by katelynienaber — CHANGES_REQUESTED** [review]
Just one change requested :)

File: packages/zowe-explorer/src/Profiles.ts [hunk 1]
```diff
@@ -283,7 +283,7 @@ export class Profiles extends ProfilesCache {
         const quickpick = vscode.window.createQuickPick();
         const placeholder = localize(
             "addSession.quickPickOption",
-            'Choose "Create new..." to define a new profile or select an existing profile to Add to the USS Explorer'
+            'Choose "Create new..." to define a new profile or select an existing profile to add to the DATA SETS Explorer'
```
[the remark this answers is not in the document]
Comment (anokhikastiaIBM) [reply packages/zowe-explorer/src/Profiles.ts:286]: Could we rephrase it to 'Create new..." to define a new profile or select an existing profile to Add to the Explorer'. We could leave out what type of profile we want. 

Or do you want it to be 'Create new..." to define a new profile or select an existing profile to Add to the DATA SETS, USS or Job Views Explorer'?

**Comment by anokhikastiaIBM** [comment]
@rudyflores It is a long message. We can see up to the USS Explorer word and not after that. We can shorten it up to make it fit into the bar.
Could we shorten it up by changing it to: 'Create new..." to define or select a profile to Add to the Explorer'. Or is that not specific enough?

File: packages/zowe-explorer/src/Profiles.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (katelynienaber) [reply packages/zowe-explorer/src/Profiles.ts:286]: I know it is sort of a big change, but could you create a text that changes based on which tree the user clicked on? I think you can use the `zoweFileProvider` to decide which tree was clicked. Imho it's important to tell the user that the profile will be added only to the one tree, and not all 3.

Probably we should shorten the whole message as well, since it doesn't fit in the box. If this suggested change is too big, I'm also ok with the shorter, generic message.

**Comment by rudyflores** [comment]
> @rudyflores It is a long message. We can see up to the USS Explorer word and not after that. We can shorten it up to make it fit into the bar.
> Could we shorten it up by changing it to: 'Create new..." to define or select a profile to Add to the Explorer'. Or is that not specific enough?

I think 'Create new... " to define or select a profile to add to the <TYPE> Explorer' would work better since we have to atleast specify whether it's Data Sets, USS, or Jobs.

**Comment by katelynienaber** [comment]
Also if we can't shorten the text, we could show it as additional information under the `+ Create a new connection` option in the dropdown list.

But this might make it look a bit busy, because the list of profiles to click on will show up after that...

File: packages/zowe-explorer/src/Profiles.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (anokhikastiaIBM) [reply packages/zowe-explorer/src/Profiles.ts]: I changed the display message based on the tree that was clicked using zoweFileProvider. I also shortened the message to fit the display box.

**Review by rudyflores — APPROVED** [review]
Looks all good to me and seems like it can be approved now! 

Integration and unit tests are all passing, I also tested the string change for all three views (DS, USS and Jobs) and the string seems to not overflow for me anymore as well. Thanks for working on this!

**Review by katelynienaber — APPROVED** [review]
Also LGTM, thank you for these changes :)
